core/src/main/scala/org/apache/spark/deploy/FaultToleranceTest.scala

```diff
@@ -113,7 +112,7 @@ private object FaultToleranceTest extends App with Logging {
     assertValidClusterState()
 
     killLeader()
-    delay(30 seconds)
+    delay(30.seconds)
```

Comment:
Yeah, I'd like to hear opinions on this. "30 seconds" seems like one of the more reasonable usages of postfix. Is it worth keeping, or trying to get rid of cases like this?